app/src/main/java/com/github/maleksandrowicz93/cqrsdemo/student/ApiResult.java

```diff
@@ -0,0 +1,32 @@
+package com.github.maleksandrowicz93.cqrsdemo.student;
+
+import com.github.maleksandrowicz93.cqrsdemo.student.enums.ResultCode;
+import com.github.maleksandrowicz93.cqrsdemo.student.enums.ResultProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Value;
+
+import java.util.Map;
+import java.util.Optional;
+
+@Value
+@Builder(access = AccessLevel.PACKAGE)
```

Comment:
te wartości `AccessLevel` warto importować statycznie, żeby łatwiej się to czytało